app_legacy/Helpers/render/comment.php

```diff
@@ -28,7 +28,10 @@ function RenderCommentsComponent(
             RenderPaginator($numComments, $count, $offset, "/comments.php?t=$articleTypeID&i=$articleID&o=");
         }
     } elseif ($numComments > count($commentData)) {
-        echo "Recent comments: <span class='smalltext'>(<a href='/comments.php?t=$articleTypeID&i=$articleID'>All $numComments</a>)</span>";
+        echo "<div>";
+        echo " <p>Recent comments</p>";
+        echo " <p class='smalltext'>(<a href='/comments.php?t=$articleTypeID&i=$articleID'>See all $numComments</a>)</p>";
```

Comment:
I think I understand this question but I want to be sure before answering. Can you rearticulate this question? Ultimately it's not wrapping on mobile with this change because it's converted from an inline-level element to two block elements. We can keep this as a single line on desktop though by using a flex layout on the parent div. Is that what you're looking for - a single line on desktop?

Reply:
Yeah.

Before:
![image](https://user-images.githubusercontent.com/32680403/226416076-fb16e05e-664c-4a7e-9414-cd0057576920.png)

After:
![image](https://user-images.githubusercontent.com/32680403/226416100-c80f4a3c-8fdc-4354-970c-089867a4c820.png)